module/PowerNSX.psm1

```diff
@@ -16618,6 +16618,84 @@ function Remove-NsxSslVpnClientInstallationPackage {
     end {}
 }
 
+
+function Remove-NsxSslVpn {
+
+    <#
+    .SYNOPSIS
+    Remove the global SSLVPN configuration of an existing NSX Edge Services
+    Gateway.
+
+    .DESCRIPTION
+    An NSX Edge Service Gateway provides all NSX Edge services such as firewall,
+    NAT, DHCP, VPN, load balancing, and high availability. Each NSX Edge virtual
+    appliance can have a total of ten uplink and internal network interfaces and
+    up to 200 subinterfaces.  Multiple external IP addresses can be configured
+    for load balancer, site‐to‐site VPN, and NAT services.
+
+    SSL VPN allows remote users to connect securely to private networks behind an
+    NSX Edge Services gateway and access servers and applications
+    in the private networks.
+
+    The Remove-NsxSslVpn cmdlet unconfigures the global SSLVPN configuration of
+    the specified Edge Services Gateway.
+
+    .EXAMPLE
+
+    PS C:\> Get-NsxEdge Edge01 | Get-NsxSslVpn | Remove-NsxSslVpn -confirm:$false
```

Comment:
Remove c:\> (gets added to the first line after .EXAMPLE only, so removing the blank line above will replace this...

Reply:
Also - probably make the single example without the -confirm (first example should be the 'safe and recommended' method :).  -confirm is used elsewhere in PowerNSX, so this probably doesn't need to be pointed out.